metapool/tests/test_sample_sheet.py

```diff
@@ -853,6 +853,49 @@ def test_validate_sample_sheet_scrubbed_names(self):
         self.assertStdOutEqual(message)
         self.assertTrue(isinstance(sheet, KLSampleSheet))
 
+    def test_validate_sample_sheet_scrubbed_project_names(self):
+        sheet = KLSampleSheet(self.good_ss)
+
+        remapper = {
+            'NYU_BMS_Melanoma_13059': "NYU's Tisch Art Microbiome 13059",
+            'Feist_11661': "The x.x microbiome project 1337"
+        }
+
+        for sample in sheet:
+            sample['Sample_Project'] = remapper.get(sample['Sample_Project'],
+                                                    sample['Sample_Project'])
+
+        sheet.Contact.Sample_Project.replace(remapper, inplace=True)
+        sheet.Bioinformatics.Sample_Project.replace(remapper, inplace=True)
+
+        obs = validate_sample_sheet(sheet)
+
+        message = (
+            'WarningMessage: The following project names were scrubbed for '
+            'bcl2fastq compatibility. If the same invalid characters are also '
+            'found in the Bioinformatics and Contacts sections those will be '
+            'automatically scrubbed too:\n'
+            "NYU's Tisch Art Microbiome 13059, The x.x microbiome project 1337"
+        )
+        self.assertStdOutEqual(message)
+        self.assertIsNotNone(obs)
+
+        scrubbed = {
+            'NYU_s_Tisch_Art_Microbiome_13059',
+            'The_x_x_microbiome_project_1337',
+            'Gerwick_6123'
```

Comment:
Gerwick_6123 isn't defined in the remapper above. Is it there as a control or because the asserts below need it to be there?

Reply:
In the remapper above I only listed two of the existing projects. I wanted to ensure this test also verfied that projects that don't need to be scrubbed aren't scrubbed at all. If you look at the sample sheet that's loaded for this test, you'll see that `Gerwick_6123` is there.

Reply:
Thanks!